coderd/database/dbmem/dbmem.go

```diff
@@ -88,6 +88,12 @@ func New() database.Store {
 	if err != nil {
 		panic(fmt.Errorf("failed to create default organization: %w", err))
 	}
+
+	_, err = q.InsertAllUsersGroup(context.Background(), defaultOrg.ID)
+	if err != nil {
+		panic(fmt.Errorf("failed to create default group: %w", err))
+	}
+
```

Comment:
dbmem doesn't have the concept of "migrations". I wonder if we should have something similar rather than chucking them here.

Reply:
I don't think "migrations" is the right concept, since we don't ever change an old dbmem into a new one, but it might make sense to have an abstraction that populates the initial contents of the database, which is what this does...